dbms/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.ibissource</groupId>
+		<artifactId>ibis-adapterframework-parent</artifactId>
+		<version>${revision}</version>
+	</parent>
+
+	<artifactId>ibis-adapterframework-dbms</artifactId>
+	<name>Ibis AdapterFramework Dbms</name>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.ibissource</groupId>
+			<artifactId>ibis-adapterframework-commons</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-core</artifactId>
+		</dependency>
```

Comment:
Is core ook nodig of alleen de API?

Reply:
Zonder core blijft dit in de console komen:
`ERROR StatusLogger Log4j2 could not find a logging implementation. Please add log4j-core to the classpath. Using SimpleLogger to log to the console...`

Reply:
Misschien dat je het dan kan toevoegen met `<scope>test</scope>`?

Reply:
Of in ieder geval een andere logging-implementatie wordt gespecificeerd direct in dat project.

Reply:
Als het niet nodig is moet het er niet in zitten. Ik snap dat het voor de tests handig is, dus dan moet de scope op test staan.
Mocht iemand deze module willen gebruiken moet hij/zij zelf de log implementatie kunnen bepalen.

Reply:
Duidelijk, dan zet ik die erin met `<scope>test</scope>` zoals Tim voorstelde